src/pkg/packaging-tools/framework.dependency.targets

```diff
@@ -40,8 +40,11 @@
       <!-- include all doc files -->
       <_docFilesToPackage Include="%(FilesToPackage.RootDir)%(FilesToPackage.Directory)**\%(FilesToPackage.FileName).xml" />
 
+      <_coreFXOverrideDocFiles Condition="'$(CoreFXOverridePath)' != ''" Include="@(FilesToPackage->'$(CoreFXOverridePath)/../../bin/docs/%(FileName).xml')" />
```

Comment:
I'd prefer if `CoreFXOverridePath` were normalized rather than having to keep track of this prop being nonstandard.

Reply:
If we normalize `CoreFXOverridePath` to the artifacts directory, we'd need a way to specify which configuration of CoreFX to use. We could just require that we use a matching configuration of CoreFX, but I don't know if that's what we want to do here.

Reply:
Wow, reading this back, I was very unclear. I only meant how this is `$(CoreFXOverridePath)/..` rather than the conventional `$(CoreFXOverridePath)..`. I guess I was in the mindset of slashes because I just wrote the comment above. I think the rest of the line is fine for this point in time. Sorry.